server/README.md

```diff
@@ -4,10 +4,13 @@
 
 ## Table of Contents
 
-- [Usage](#usage)
-  - [How to start the server locally](#how-to-start-the-server-locally)
-  - [Development commands](#development-commands)
-- [Architecture](#architecture)
+- [TheHaloMod-SPA Server](#thehalomod-spa-server)
+  - [Table of Contents](#table-of-contents)
+  - [Usage](#usage)
+    - [How to start the server locally](#how-to-start-the-server-locally)
+    - [Development commands](#development-commands)
+    - [Production deployment using docker](#production-deployment-using-docker)
+  - [Architecture](#architecture)
```

Comment:
The top level `TheHaloMod-SPA Server` probably doesn't need to be included in the table of contents, or table of contents itself. But that is a small thing and won't block this from being approved.

Reply:
Agreed